internal/transports/p2p/peer/utils.go

```diff
@@ -0,0 +1,44 @@
+package peer
+
+import (
+	"github.com/bitcoin-sv/block-headers-service/internal/chaincfg"
+	"github.com/bitcoin-sv/block-headers-service/internal/wire"
+)
+
+func minUint32(a, b uint32) uint32 {
+	if a < b {
+		return a
+	}
+	return b
+}
```

Comment:
As far as I recall - golang introduced similar function.
https://www.linkedin.com/pulse/built-in-min-max-functions-go-121-robert-gogolok/

Reply:
I thought it's not gonna work with uint32, but it does, thanks :)